.gitmodules

```diff
@@ -1,8 +1,6 @@
 [submodule "libchewing"]
 	path = libchewing
 	url = https://github.com/chewing/libchewing.git
-	branch = release/0.8.x
```

Comment:
Just curious, so it is not going to track stable releases anymore?
Personally I like this change.

Reply:
By default the submodule still tracks the stable release, but when using `update --remote` it will fetch the latest default branch.

I made this change so that we can build both release builds and nightly builds from the same workflow.

Reply:
>I made this change so that we can build both release builds and nightly builds from the same workflow.

👍